src/main/java/frc/robot/commands/TargetNote.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Constants.SwerveModuleConstants;
+import frc.robot.subsystems.DriveSubsystem;
+import frc.robot.subsystems.Limelight;
+
+public class TargetNote extends Command {
+  private DriveSubsystem m_drive;
+  private Limelight m_limelight;
+  private ProfiledPIDController thetaController = new ProfiledPIDController(
+      SwerveModuleConstants.kPTurningController,
+      SwerveModuleConstants.kITurningController,
+      SwerveModuleConstants.kDTurningController,
+      new TrapezoidProfile.Constraints(
+        SwerveModuleConstants.kMaxAngularSpeedRadiansPerSecond,
+        SwerveModuleConstants.kMaxAngularAccelerationRadiansPerSecondSquared));
```

Comment:
I reused the constants from the swerve module. Not sure if they should be different.

Reply:
Using the same constants should be fine. We probably don't need/want the robot to turn any faster/slower when tracking a note than when under manual control.